styles/custom.scss

```diff
@@ -81,7 +81,7 @@ $theme-colors: (
   }
 }
 
-@import "node_modules/bootstrap/scss/bootstrap";
+@import "~bootstrap/scss/bootstrap";
```

Comment:
trying to fix this [build error](https://app.netlify.com/sites/atd-product/deploys/6188390a2778b80008032614), which i can't reproduce locally :/